src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1508,30 +1508,29 @@ private static LambdaConversionResult IsAnonymousFunctionCompatibleWithDelegate(
                 // SPEC: and modifiers as the corresponding parameter in F.
                 // SPEC: If F has an implicitly typed parameter list, D has no ref or out parameters.
```

Comment:
>If F has an implicitly typed parameter list, D has no ref or out parameters. [](http://example.com/codeflow?start=25&length=76)

Is this still true? If not, let's remove this line.

Reply:
good catch.  fixed.